.github/workflows/deploy-bridgedigitalhealth-tst.yaml

```diff
@@ -0,0 +1,16 @@
+name: deploy-arc-tst
+
+on:
+  push:
+    branches: ["staging"]
+
+jobs:
+  deploy:
+    permissions:
+      id-token: write
+      contents: read
+    uses: ./.github/workflows/build-and-deploy.yaml
+    with:
+      branch: ${{ github.ref_name }}
+      destination_bucket: tst-bridgedigitalhealth-websitebucket-1fnzdxhja8yl4
```

Comment:
Just curious, which domain name was configured for the open-bridge staging site? Long-term, will we set up another stack for the production open-bridge site?

Reply:
We will have staging.bridgedigital.health and prod.bridgedigital.health (also configured as bridgedigital.health). I already deployed infra a tst.bridgedigital.health to test the PRs in the infra (which is why I'm targeting tst everywhere here).